gettime.c

```diff
@@ -559,12 +559,14 @@ struct clock_entry {
 	uint64_t tsc;
 };
 
+static pthread_mutex_t clock_lock;
```

Comment:
This might be miserable on large CPU counts, where the previous approach used private mutexes. I still think the best solution, if the cross-thread lock/unlock is an issue, is to switch them to fio_mutex'es.